src/components/BlogList/BlogList.css

```diff
@@ -0,0 +1,120 @@
+.blog-container {
+  width: 85%;
+  display: flex;
+}
+.blog-container .blog-list {
+  width: 40%;
+  background-color: #f5f6fa;
+  padding: 30px;
+}
+.blog-container .blog-list .blog-list-header {
+  display: flex;
+  align-items: normal;
+  justify-content: space-between;
+  gap: 15px;
+  margin-bottom: 20px;
+}
+.blog-container .blog-list .blog-list-header .search-input {
+  width: 75%;
+  height: 50px;
+  padding: 0 20px;
+  border-radius: 28px;
+  border: none;
+  background-color: #ebecf3;
```

Comment:
Colors can be moved into SCSS variables

Reply:
Flex styles and colors are added. Please look into scss files.